src/plugins/template/tests/functional/skip_tests_config.cpp

```diff
@@ -118,7 +118,10 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*ReferenceInterpolate_v11.*data_type=f16.*)",
         R"(.*LSTMSequence_With_Hardcoded_Refs.*ReferenceLSTMSequenceTest.*iType=f16.*)",
         // CVS-111443
-        R"(.*eltwiseOpType=Mod_secondaryInputType=PARAMETER_opType=VECTOR_NetType=(f16|f32).*)"
+        R"(.*eltwiseOpType=Mod_secondaryInputType=PARAMETER_opType=VECTOR_NetType=(f16|f32).*)",
+        // CVS-94156 AUTO and MULTI does not support
```

Comment:
Create a separate ticket to track AUTO/MULTI issue, or fix the issue here.

Reply:
at least we need to understand why these tests should be disabled (even temporary)